bus-mapping/src/circuit_input_builder.rs

```diff
@@ -813,13 +820,21 @@ pub fn get_dummy_tx_hash() -> H256 {
     H256(tx_hash)
 }
 
+/// Get the dummy hash
+pub fn get_dummy_l1_block_hash() -> H256 {
+    Hash::from_str("5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6").unwrap()
+}
```

Comment:
I will recommend using `LazyStatic` or `OnceCell` to declare this, as its a constant.